mobile/src/modal.js

```diff
@@ -43,14 +42,14 @@ export class CustomModal extends Component {
             fontSize: 18,
             color: '#9B9B9B',
             textAlignVertical: 'top',
-            height: this.state.height,
             maxHeight: 100,
-            marginTop: this.state.marginTop
+            height: this.state.height,
+            marginTop: 20,
+            paddingTop: 0,
         }
         
-        const androidStyle = {
-          paddingTop: 2,
-          paddingLeft: 0
+        const androidStyle1 = {
```

Comment:
is this name change a typo?

Reply:
Was testing 2 different style constants. Will update.